features/news_api_item.feature

```diff
@@ -113,9 +113,12 @@ Feature: News API Item
       "body_html": "<p>test&nbsp;test</p>",
       "associations": {
         "featuremedia": {
+          "products": ["12345", "6789"],
```

Comment:
You could also add to the response of `GET "/news/item/#items._id#?format=NINJSFormatter2"`
```
"products": "__no_value__",
"subscribers": "__no_value__"
```
To make sure these keys don't exist in the response